pm_symmetry/pm_symmetry.xml

```diff
@@ -0,0 +1,40 @@
+<tool id="pm_symmetry" name="Generate Wyckoff points symmetry report (pm-symmetry)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">pymuonsuite</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip3 show pymuonsuite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        ln -s $structure input.cell &&
+        pm-symmetry input.cell > out.txt 
+    ]]></command>
+    <inputs>
```

Comment:
Same comments as usual for inputs/outputs: clear names, labels and help text please. 

Remember that the `name` can appear in the workflow UI so it's not totally hidden from the user - 'structure' is a fine name, but 'out_sym' isn't that clear.

Reply:
Structure parameter needs a label and some help text still - can be taken from pm_muairss_write